content/_data/changelogs/weekly.yml

```diff
@@ -915,6 +915,47 @@
     # pull: 2960 not notable enough
     # pull: 2963 not notable enough
     # pull: 2964 not notable enough
+- version: "2.74"
+  date: 2017-08-13
+  changes:
+    - type: major rfe
+      message: >
+        Upgrade Stapler library from 1.250 to 1.252
+      references:
+        - pull: 2956
+        - url: https://github.com/stapler/stapler/blob/master/CHANGELOG.md
+          title: Stapler changelog
+    - type: rfe
+      message: >
+        Improve Polish localization.
+      pull: 2974
+    - type: bug
+      message: >
+        Prevent caching of the items list by the browser to prevent staleness.
+      pull: 2973
+      issue: 43848
+    - type: bug
+      message: >
+        Show display name of the current view in window title.
+      # Improvement too minor: Only use the special 'Dashboard' label outside any folder
+      pull: 2969
+    - type: bug
+      message: >
+        Include culprits in XML and JSON API again (regression in 2.60).
+      issue: 46082
+      pull: 2978
+    - type: bug
+      message: >
+        Improve robustness of the reverse build trigger ("Build after other projects are built").
```

Comment:
Where is "dependency tree" exposed in the Jenkins UI, so that users are aware what that even is?